100nocks166ryoryo/CSS/06/float.css

```
@@ -0,0 +1,33 @@
+#aka { float: left;
```

05/new.cssのように
```suggestion
#aka { 
   float: left;
```
ブラケットのあとは改行しましょうか〜